code/modules/mining/machine_vending.dm

```diff
@@ -213,7 +213,7 @@
 /obj/machinery/mineral/equipment_vendor/golem/Initialize()
 	desc += "\nIt seems a few selections have been added."
 	prize_list += list(
-		new /datum/data/mining_equipment("Extra Id",       				/obj/item/card/id/mining, 				                   		250),
+		new /datum/data/mining_equipment("Extra Id",       				/obj/item/card/id/advanced/mining,								250),
```

Comment:
```suggestion
		new /datum/data/mining_equipment("Extra Id", /obj/item/card/id/advanced/mining, 250),
```
I swear, this pattern of adding a million spaces is diff hell.
Can you just replace these with a space each? Or just accept the suggestion if you don't want to go there.